requirements.txt

```diff
@@ -1,3 +1,4 @@
+Jinja2==3.1.6
```

Comment:
Here and down below, if not needed I'd avoid fixing the versions, that why we use the lock file. If you want to ensure compatibility then use `~=`.